build.gradle

```diff
@@ -31,7 +31,7 @@ buildscript {
         
         // Scala plugins
         classpath "com.adtran:scala-multiversion-plugin:1.0.35"
-        classpath "com.diffplug.spotless:spotless-plugin-gradle:3.24.2"
+        classpath "com.diffplug.spotless:spotless-plugin-gradle:3.27.1"
```

Comment:
Upgrading the spotless plugin removes the Gradle deprecation warnings that were generated by the earlier version.

Reply:
I thought you meant that you were handling this in a separate ticket.

Reply:
Do we use spotbugs plugin anymore?  I'm confused about the relationship between these two plugins.

Reply:
Currently, the spotbugs plugin is used in the java projects and the spotless plugin is used in the scala projects.